prow/plugins.yaml

```diff
@@ -592,6 +592,9 @@ plugins:
   - release-note
   - require-matching-label
   - welcome
+  
+  kubernetes-sigs/cluster-api-provider-ibmcloud:
+  - milestone
```

Comment:
You probably want to configure the plugin to restrict `/milestone` to some other team than the default of `@kubernetes/kubernetes-milestone-maintainers`.  Please see how eg: kubernetes-sigs/cluster-api-provider-aws did it further above in this file.  GitHub teams are created/managed by the use of PR's against the kubernetes/org repo